guide/deployment/README.md

```diff
@@ -0,0 +1,52 @@
+---
+{
+    'tableData':
+        {
+            'docker':
+                {
+                    'eou': 'Hard learning curve',
+                    'perf': 'How Docker works under the hood is by utilizing containers which are basically little light virtual machines, so expect a bit of overhead if your VPS is on the low-end.',
+                    'update': "Updating your bot is easy with Docker. Simply setup [Watchtower](https://containrrr.dev/watchtower/) to monitor your bot's Docker container and update it when it notices a new Image.",
+                    'devops': 'Working with Docker also gives you the ability to easily integrate your bot with other services, such as [GitHub Actions](#automating-the-build-process-with-github-actions) for example.',
+                },
+        },
+}
+---
+
+# Deployment
+
+Congratulations! 🎉 On making it this far, now it's time to deploy your bot to production. 🚀
```

Comment:
```suggestion

```
unnecessary, does not match the style of the guide

Reply:
It's an important event in the average lifecycle of a bot dev.